scripts/Run-Tests.ps1

```diff
@@ -43,7 +43,7 @@ $TestRunnerRootPath = "$NuGetPackageRoot\xunit.runner.console\2.3.1\tools\"
 foreach ($project in $Projects.NewTest) {
     foreach ($framework in $Frameworks.Application) {
         Write-Information "Running tests in ${project}: $framework..."
-        Push-Location $BinRoot\$project\AnyCPU_$Configuration\$framework
+        Push-Location $BinRoot\AnyCPU_$Configuration\$project\$framework
```

Comment:
>AnyCPU [](start = 31, length = 6)

Why aren't we using the $(Platform) var here? Oversight?

Reply:
Really just oversight, I assume--this was how it was originally.  I'll change both to platform.